pkg/accessibility/accessibility.go

```diff
@@ -0,0 +1,33 @@
+package accessibility
+
+import (
+	"os"
+
+	"github.com/cli/go-gh/v2/pkg/config"
+)
+
+// ACCESSIBILITY_ENV is the name of the environment variable that can be used to enable
+// accessibility features. If the value is empty, "0", or "false", the accessibility
+// features are disabled. Any other value enables the accessibility features. Note that
+// this environment variable supercedes the configuration file's accessible setting.
+const ACCESSIBILITY_ENV = "ACCESSIBILE"
```

Comment:
This is spelt incorrectly. "ACCESSIBILE" -> "ACCESSIBLE"

Reply:
Fixed in https://github.com/cli/go-gh/pull/182/files#r1954881696